src/mod_mbedtls.inl

```diff
@@ -42,6 +45,68 @@ static void mbed_debug(void *context,
                        const char *str);
 static int mbed_ssl_handshake(mbedtls_ssl_context *ssl);
 
+#if defined(CONFIG_ENTROPY_HAS_DRIVER)
+static const struct device *entropy_dev;
+#endif
+
+
+#if defined(CONFIG_ENTROPY_HAS_DRIVER)
+#if !defined MAX_RANDOMNESS_JUNK_SIZE
+#define MAX_RANDOMNESS_JUNK_SIZE 1024
+#endif
+static int tls_entropy_func(void *unused, unsigned char *output, size_t len)
```

Comment:
Do I understand correctly that it's same as `true_randomness_generate` from `true_randomness`?

Reply:
Yes I basically copied it over from that function and adjusted the backup function for qemu a bit.